### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by gscagl** [idiydownunder/binance-trader-bot#1]
[06/08/24 16:33:01] [ERR] Unknown Error: 'maxQty' 

What does it means?

**Comment by frihauge** [comment]
Yes i also followed the install procedure and ended up with the: 
 [ERR] Unknown Error: 'maxQty'

**Comment by 71ae** [comment]
As I'm having the same problem, I've submitted a fix which works for me:
https://github.com/idiydownunder/binance-trader-bot/pull/2
